api_tests/e2e/rfc003/btc_eth/happy.ts

```diff
@@ -2,283 +2,153 @@ import * as bitcoin from "../../../lib/bitcoin";
-        let body = (await alice.poll_comit_node_until(
-            alice_swap_href,
-            body => body._links.redeem
-        )) as SwapResponse;
-        let alice_redeem_href = body._links.redeem.href;
-        let res = await chai
-            .request(alice.comit_node_url())
-            .get(alice_redeem_href);
-        res.should.have.status(200);
-        alice_redeem_action = res.body;
-    });
-
-    let alice_eth_balance_before: BN;
-
-    it("[Alice] Can execute the redeem action", async function() {
-        alice_redeem_action.payload.should.include.all.keys(
-            "contract_address",
-            "data",
-            "amount",
-            "gas_limit",
-            "network"
-        );
-        alice_eth_balance_before = await ethereum.ethBalance(
-            alice_final_address
-        );
-        await alice.do(alice_redeem_action);
-    });
-
-    it("[Alice] Should have received the beta asset after the redeem", async function() {
-        let alice_eth_balance_after = await ethereum.ethBalance(
-            alice_final_address
-        );
-
-        let alice_eth_balance_expected = alice_eth_balance_before.add(
-            beta_asset_quantity
-        );
-
-        alice_eth_balance_after
-            .eq(alice_eth_balance_expected)
-            .should.be.equal(true);
-    });
-
-    let bob_redeem_action: Action;
-
-    it("[Bob] Can get the redeem action after Alice redeems", async function() {
-        this.timeout(10000);
-        let body = (await bob.poll_comit_node_until(
-            bob_swap_href,
-            body => body._links.redeem
-        )) as SwapResponse;
-        let bob_redeem_href = body._links.redeem.href;
-        let res = await chai
-            .request(bob.comit_node_url())
-            .get(
-                bob_redeem_href +
-                    "?address=" +
-                    bob_final_address +
-                    "&fee_per_byte=20"
-            );
-        res.should.have.status(200);
-        bob_redeem_action = res.body;
-    });
-
-    it("[Bob] Can execute the redeem action", async function() {
-        bob_redeem_action.payload.should.include.all.keys("hex", "network");
-
-        await bob.do(bob_redeem_action);
-        await bitcoin.generate();
-    });
-
-    it("[Bob] Should have received the alpha asset after the redeem", async function() {
-        this.timeout(10000);
-        let body = (await bob.poll_comit_node_until(
-            bob_swap_href,
-            body => body.state.alpha_ledger.status === "Redeemed"
-        )) as SwapResponse;
-        let bob_redeem_txid = body.state.alpha_ledger.redeem_tx;
-
-        let bob_satoshi_received = await bitcoin.getFirstUtxoValueTransferredTo(
-            bob_redeem_txid,
-            bob_final_address
-        );
-        const bob_satoshi_expected = alpha_asset_quantity - alpha_max_fee;
-
-        bob_satoshi_received.should.be.at.least(bob_satoshi_expected);
-    });
-});
+test();
```

Comment:
:wrench: Instead of having to name to this function, you could use the [IIFE pattern](https://developer.mozilla.org/en-US/docs/Glossary/IIFE).